bigquery/indexer.py

```diff
@@ -179,11 +119,11 @@ def main():
     args = parse_args()
 
     json_path = os.path.join(args.dataset_config_dir, 'dataset.json')
```

Comment:
Spawned off of my other comment here: this function could be a bit easier to read if we declared the variables where they're used (e.g. move declaration of `primary_key` down to line 132). Seems like most programmers agree that this makes for more readable code than declaring all variables in a block at the top of a function/program

Reply:
Done